analytics/spark/src/main/java/org/locationtech/geowave/analytic/spark/GeoWaveRDDLoader.java

```diff
@@ -134,4 +136,59 @@ public static GeoWaveIndexedRDD loadIndexedRDD(
 
     return javaRdd;
   }
+
+  public static JavaPairRDD<GeoWaveInputKey, GridCoverage> loadRawRasterRDD(
+      final SparkContext sc,
+      final DataStorePluginOptions storeOptions,
+      final String indexName,
+      final Integer minSplits,
+      final Integer maxSplits) throws IOException {
+    if (sc == null) {
+      LOGGER.error("Must supply a valid Spark Context. Please set SparkContext and try again.");
+      return null;
+    }
+
+    if (storeOptions == null) {
+      LOGGER.error("Must supply input store to load. Please set storeOptions and try again.");
+      return null;
+    }
+
+    final Configuration conf = new Configuration(sc.hadoopConfiguration());
+
+    GeoWaveInputFormat.setStoreOptions(conf, storeOptions);
+
+    if (indexName != null) {
+      GeoWaveInputFormat.setQuery(
+          conf,
+          QueryBuilder.newBuilder().indexName(indexName).build(),
+          storeOptions.createAdapterStore(),
+          storeOptions.createInternalAdapterStore(),
+          storeOptions.createIndexStore());
+    }
+    if (((minSplits != null) && (minSplits > -1)) || ((maxSplits != null) && (maxSplits > -1))) {
```

Comment:
Should this be `&&` instead of `||`, or is it fine to specify only `minSplits` or `maxSplits`?